how-to/how-to-use-rootless-kata-with-podman.md

```diff
@@ -0,0 +1,108 @@
+# How to use Kata Containers with Podman rootlessly
+
+For an even more secure system,
+[Kata Containers](https://Katacontainers.io) can run workloads without a privileged user. Using <br>
+[Podman](https://podman.io/) as the container engine, and
+[slirp4netns](https://github.com/rootless-containers/slirp4netns) for user-space networking
+
+## Requirements
+- A Linux system, see [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions) for an updated list
+  - If using Centos7 follow [this guideline](https://www.scrivano.org/2018/10/12/rootless-podman-from-upstream-on-centos-7/) to correctly setup namespaces (*This should be fixed in Rhel 7.6*).
+- Golang 1.12
+
+## Installation
+ 
+To install Podman view [Installation instructions](https://github.com/containers/libpod/blob/master/install.md)<br>
+To install Kata-Containers view the [installation guidelines](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+The following are the minimum required versions:
+* **Podman-xx.xx.xx**
+* **slirp4netns-0.4.0**
+* **Kata-Containers-[runtime](https://github.com/kata-containers/runtime)-1.9.0**
+
+## Configuration
+Now that Kata Containers and Podman have been installed, they need to be configured for rootless execution
+
+#### 1. Disable SELinux
+If enabled, disable SELinux. Kata Containers does not support SELinux; it is a known [limitation](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)
+<br>*Note: this command may differ depending on the distro being used*
```

Comment:
Agreed. I think this should be a **Warning:** as this step requires users to disable a security feature... It would be good to know *why* this is required. Have we investigated creating a new security policy to avoid having to disable this entirely? Maybe podman are working on this?

Reply:
AFAIK this is a Kata issue, and not a Podman issue: https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support